src/org/opendatakit/briefcase/model/BriefcaseAnalytics.java

```diff
@@ -55,6 +57,14 @@ public void trackEvent(String action, String label, Integer value) {
      * Track this application's startup event.
      */
     public void trackStartup() {
-        trackEvent("Application-Startup", null, null);
+        trackEvent("Application-Startup", null, null, false);
     }
-}
+
+    /**
+     * Tracks the user giving/withdrawing their consent for usage tracking, and saves the new preference.
+     */
+    public void trackConsentDecision(boolean givesConsent) {
+        BriefcasePreferences.setBriefcaseTrackingConsentProperty(givesConsent);
+        trackEvent(givesConsent ? "Tracking-Enabled" : "Tracking-Disabled", null, null, true);
+    }
+}
```

Comment:
Will do, I did a quick Google reveals that it seems to have an affect on some of Unix's tools because they expect all valid lines to end in a newline.